.circleci/images/runner/Dockerfile

```diff
@@ -7,7 +7,8 @@ RUN set -ex \
         gcc g++ make git ssh curl pkg-config file \
         python-dev python-setuptools python-pip \
         python3.7-dev python3-distutils python3-pip python3-yaml \
-        libssl-dev libsnmp-base libsnmp-dev libpq-dev snmp-mibs-downloader libsystemd-dev
+        libssl-dev libsnmp-base libsnmp-dev libpq-dev snmp-mibs-downloader libsystemd-dev \
+        doxygen graphviz
```

Comment:
why is graphviz also needed?

Reply:
`dot` (included in the `graphviz` package) is needed to draw the class & dependency graphs  on each page of the documentation. It's not strictly required (the `doxygen` command will still complete), but not adding it will make doxygen output a bunch of error / warning messages because of `dot` not being available.

Reply:
👍